tests/end-to-end/external-solution-integration/internal/scenario/connectivity_adapter/scenario.go

```diff
@@ -12,13 +14,15 @@ type Scenario struct {
 	domain        string
 	testID        string
 	skipSSLVerify bool
+	waitTime      time.Duration
```

Comment:
I feel like `waitTime` is to general term and it is only used before sending the event. I would suggest renaming it to something more descriptive like `eventSendDelay`.

Reply:
Yes, it's more general, but it's used to make the general step:
```
testsuite.NewSleep(s.waitTime)
```
configurable. The name is the same as in the other e2e tests.
We have done different tuning attempts to use the ```Sleep test step``` after other KN related actions too, but remained now at this simple solution. It could be not the last one... The main idea is to have similar code related to KN in all our three e2e test scenarios. The main issue is here that no API exists in KN/Eventing (like a "ready status")  which reflect the readiness of the eventing related infrastructure.

Reply:
Done